app/views/plugins/_show.haml

```diff
@@ -0,0 +1,19 @@
+- runtimeDiv = "output-#{plugin.id}"
+= content_for :external_scripts do
+  %script{src:plugin.url}
+
+.output{id:runtimeDiv}
```

Comment:
Could we render that in some reasonable place and make sure it's not hidden? I guess plugin-output could be a more descriptive class name too.
E.g. I could imagine that to be at the end of the activity, so a plugin can render some status / output without much effort (even for debugging).

Reply:
Yes -- For sure.  I was just copying the pattern we had as a first pass.  I  will be making further commits to improve this.